Kickstarter-iOS/Views/Controllers/ThanksViewController.swift

```diff
@@ -107,12 +107,6 @@ internal final class ThanksViewController: UIViewController, UITableViewDelegate
 
     self.backedLabel.rac.attributedText = self.viewModel.outputs.backedProjectText
 
-    self.viewModel.outputs.dismissToRootViewController
-      .observeForControllerAction()
-      .observeValues { [weak self] in
-        self?.view.window?.rootViewController?.dismiss(animated: true, completion: nil)
```

Comment:
Hey I'm still not entirely sure what the problem was and why the workaround was required. I found that by changing this line to `self?.dismiss(animated: true)` it achieves what we want? Let me know if I'm missing something!

<img src="https://user-images.githubusercontent.com/3735375/69892579-b7cbea00-12bb-11ea-8e96-80871b3ede01.gif" width="50%"/>